watertap/examples/flowsheets/full_treatment_train/analysis/tests/test_multi_sweep.py

```diff
@@ -0,0 +1,50 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+import pytest
+import os
+from pyomo.environ import value
+from watertap.examples.flowsheets.full_treatment_train.analysis import (flowsheet_NF,
+                                                               flowsheet_NF_no_bypass,
+                                                               flowsheet_single_stage,
+                                                               flowsheet_two_stage,
+                                                               flowsheet_NF_two_stage,
+                                                               flowsheet_softening,
+                                                               flowsheet_softening_two_stage)
+from watertap.examples.flowsheets.full_treatment_train.analysis.multi_sweep import *
+from watertap.tools.parameter_sweep import _init_mpi, LinearSample, parameter_sweep
+
+pytest_parameterize_list = []
+for case_num in [1, 2, 3, 4, 8, 9]:
+    for ro_type in ['0D', '1D']:
+        test_case = (case_num, ro_type)
+        pytest_parameterize_list.append(test_case)
+
+@pytest.mark.parametrize('case_num, RO_type', pytest_parameterize_list)
```

Comment:
This seems to best fit the theme of an integration test.
```suggestion
@pytest.mark.parametrize('case_num, RO_type', pytest_parameterize_list)
@pytest.mark.integration
```